### PR TITLE
Support timing budgets

### PR DESCRIPTION
**Pull request by starrett67**

### CODE REVIEW

**Comment by andreasonny83** [comment]
Hi @starrett67 .
I would be happy to merge in your PR if you can just format the file correctly so it can pass the tests.
Please run `npm run format` then make sure all the tests pass with `npm test` and commit your new changes.
Thank you

**Comment by starrett67** [comment]
> Hi @starrett67 .
> I would be happy to merge in your PR if you can just format the file correctly so it can pass the tests.
> Please run `npm run format` then make sure all the tests pass with `npm test` and commit your new changes.
> Thank you

Sorry about that. I opened this while trying to figure out how we could do this in a pair programming session. I opened the PR so I could remember to go back and properly format it. I'll get that done sometime today.

File: lib/lighthouse-reporter.js [hunk 1]
```diff
@@ -132,6 +132,8 @@ const createBudgetsReport = (results) => {
       results.audits['performance-budget'].details.items) ||
     [];
 
+  const timings = results?.audits?.['timing-budget']?.details?.items || [];
```
Comment (starrett67) [inline lib/lighthouse-reporter.js:135]: Should I refactor this part so that users dont need to have node14?

Comment (andreasonny83) [reply lib/lighthouse-reporter.js:135]: maybe makes sense actually

**Review by andreasonny83 — APPROVED** [review]
All good. Thanks a lot

**Pull request merged**